diff --git a/macro.js b/macro.js
index 4df3979dfc9b730e7be8dbcc7e702b09699fbaa1..e4fbe71a5e4bf1478ff18491b1e1961b3930048f 100644
--- a/macro.js
+++ b/macro.js
@@ -1774,7 +1774,8 @@ function getJsxAttributes(path) {
   return attributes.filter(a => a.isJSXAttribute());
 }
 
-const validImports = new Set(['default', 'styled', 'css', 'theme', 'screen', 'TwStyle', 'TwComponent', 'ThemeStyle', 'GlobalStyles', 'globalStyles']);
+// add stylable to `twin.macro` namespace, see `{apps/frontend}|{pkgs/ui}/types/twin.d.ts`
+const validImports = new Set(['default', 'stylable', 'styled', 'css', 'theme', 'screen', 'TwStyle', 'TwComponent', 'ThemeStyle', 'GlobalStyles', 'globalStyles']);
 function validateImports(imports, coreContext) {
   const importTwAsNamedNotDefault = Object.keys(imports).find(reference => reference === 'tw');
   coreContext.assert(!importTwAsNamedNotDefault, ({